fido/src/main/java/com/yubico/yubikit/fido/client/BasicWebAuthnClient.java

```diff
@@ -476,14 +470,14 @@ protected Ctap2Session.CredentialData ctapMakeCredential(
                     getCredentialList(excludeCredentials),
                     null,
                     ctapOptions.isEmpty() ? null : ctapOptions,
-                    pinUvAuthParam,
-                    pinUvAuthProtocol,
+                    authParams.pinUvAuthParam != null ? authParams.pinUvAuthParam.clone() : null,
```

Comment:
Should we really make a copy to send in to ctapMakeCredential here? It seems to me that the original object should be used.